src/Native/Runtime/inc/eetype.h

```diff
@@ -444,6 +444,11 @@ class EEType
     bool IsGeneric()
         { return (m_usFlags & IsGenericFlag) != 0; }
 
+#if defined(CORERT)
+    ModuleManager* GetModule()
```

Comment:
`GetModuleManager` may be a better name. Also, could you please change the one existing places that does `*m_ppModuleManager` to call this method instead?